imports/plugins/core/orders/client/templates/workflow/cancelOrder.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-undef */
+
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+
+const validateComment = (comment) => {
+  check(comment, Match.OptionalOrNull(String));
+
+  // Valid
+  if (comment !== "Select One") {
+    return true;
+  }
+
+  // Invalid
+  return {
+    error: "INVALID_COMMENT",
+    reason: "Select a reason for canceling order"
+  };
+};
+
+Template.coreOrderCancelOrder.onCreated(function () {
```

Comment:
We want to be consistent with the use of `arrow functions` vs `anonymous function`